### ISSUE TITLE
Remove user-journeys legacy content

### ISSUE THREAD

**Issue by zacharysarah** [kubernetes/website#18615]
## What's the bug?

The docs' former User Journeys content no longer appears in site navigation, but:

- The files are still in the repository
- The files still build to orphaned content on the website

For example: https://kubernetes.io/docs/user-journeys/users/application-developer/advanced/

## What's the solution?

Delete the user journeys tree:

https://github.com/kubernetes/website/tree/master/content/en/docs/user-journeys

## TLDR

![delete](https://user-images.githubusercontent.com/3210446/72186981-582d2800-33ab-11ea-9656-90e84bd5477e.gif)

/sig docs
/honk

**Comment by gavinfish** [comment]
/assign

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by gavinfish** [comment]
Let's save it for a new contributor:smile:.
/unassign

**Comment by zacharysarah** [comment]
Closed by #18779 
/close

**Comment by gavinfish** [comment]
/reopen
There should be more follow ups here.
